src/subscript/field_statistics/field_statistics.py

```diff
@@ -981,37 +1277,54 @@ def main():
     zone_code_names = field_stat["zone_code_names"]
     facies_per_zone = get_facies_per_zone(GLOBAL_VARIABLES_FILE)
     result_path = RESULT_PATH
+    stat_list= ["mean", "stdev"]
+    iter_list = field_stat["iterations"]
     if "use_zones" in field_stat:
         zone_list= field_stat["use_zones"]
     else:
         zone_list= list(zone_code_names.values())
-    stat_list= ["mean", "stdev"]
-    iter_list = field_stat["iterations"]
-
     cont_prop_dict = field_stat["continuous_property_param_per_zone"]
 
     discrete_prop_dict = field_stat["discrete_property_param_per_zone"]
 
     label = LABEL
-    for stat in stat_list:
-        for zone in zone_list:
-            for iteration in iter_list:
-                if cont_prop_dict:
-                    if zone in cont_prop_dict:
-                        for prop_name in cont_prop_dict[zone]:
-                            name = \
-                            "ertbox--" + stat + "_" + zone + "_" + prop_name \
-                            + "_" + str(iteration)
+    for zone in zone_list:
+        if cont_prop_dict:
+            if zone in cont_prop_dict:
+                for stat in stat_list:
+                    for prop_name in cont_prop_dict[zone]:
+                        for iteration in iter_list:
+                            name =  \
+                            "ertbox--" + stat + "_" + zone \
```

Comment:
As a future todo: use `fmu-dataio` to export these. Currently, these results will not be availbale in SUMO.

Reply:
OK. Tell me how to do that offline..